src/main/fc/tuning_sliders.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "flight/pid.h"
+
+typedef enum {
+    SLIDER_DISABLED = 0,
+    SLIDER_ENABLED
+} tuningSliderState_e;
```

Comment:
This doesn't seem to be used anywhere - and it shouldn't, a `bool` will do just fine for bi-state variables.

Reply:
It is used in `applyTuningSliders()` and in reset config as a default value. But I'll change it as you say, it is unnecessary.